arm/Microsoft.ManagedIdentity/userAssignedIdentities/.bicep/nested_rbac.bicep

```diff
@@ -0,0 +1,12 @@
+param roleAssignment object
+param builtInRoleNames object
+param userMsiName string
+
+resource nested_rbac 'Microsoft.ManagedIdentity/userAssignedIdentities/providers/roleAssignments@2020-04-01-preview' = [for i in range(0, length(roleAssignment.principalIds)): {
```

Comment:
As noticed by @MariusStorhaug it could be beneficial to reference to the nested_rbac version for resource groups (cleaner code)

Reply:
Not reference implement here. We do not (yet) cross-reference modules

Reply:
See below comment